.stylelintrc.js

```diff
@@ -2,12 +2,18 @@ module.exports = {
   extends: ['stylelint-config-wolox'],
   rules: {
     'selector-max-type': 2,
-    'at-rule-no-unknown': [ true, {
-      ignoreAtRules: ['if', 'for', 'extend', 'mixin']
-    }],
-    'scss/no-duplicate-dollar-variables': [ true, {
-      ignoreInsideAtRules: ['if', 'mixin', 'media']
-    }],
-    'scss/at-import-no-partial-leading-underscore': null,
+    'at-rule-no-unknown': [
+      true,
+      {
+        ignoreAtRules: ['if', 'for', 'extend', 'mixin']
+      }
+    ],
+    'scss/no-duplicate-dollar-variables': [
+      true,
+      {
+        ignoreInsideAtRules: ['if', 'mixin', 'media']
+      }
+    ],
+    'scss/at-import-no-partial-leading-underscore': null
```

Comment:
please avoid modifying configuration files this may affect other developers when working in a team, however you can modify them on your machine and not upload the changes unless you are sure the changes are necessary for other machines